.github/workflows/actions/checkout-prestashop/action.yml

```diff
@@ -58,6 +58,6 @@ runs:
 
     # Workaround until https://github.com/PrestaShop/PrestaShop/issues/29813 is fixed
     - name: PrestaShop Configuration (Copy of Config API)
-      if: (inputs.base_branch == '8.1.x' || inputs.base_branch == '8.2.x')
+      if: (inputs.base_branch == '8.1.x' || inputs.base_branch == '8.2.x' || inputs.base_branch == '9.0.x' )
```

Comment:
```suggestion
      if: (inputs.base_branch == '8.1.x' || inputs.base_branch == '8.2.x')
```

Reply:
yep its only needed for v8, so not develop nor 9.0